blkn/src/Button/index.js

```diff
@@ -1,22 +1,55 @@
 // @flow
 import * as React from "react";
+import Icon from "./Icon";
 
 type Props = {
   title: string,
-  onClick: (e: SyntheticEvent<HTMLButtonElement>) => void,
+  onClick?: (e: SyntheticEvent<HTMLButtonElement>) => void,
+  isDisabled?: boolean,
+  type: "primary" | "secondary",
+  size: "normal" | "large" | "small",
+  icon?: Function,
 };
 
-const Button = ({ title, onClick }: Props) => (
-  <button onClick={onClick}>
-    {title}
+const paddings = {
+  normal: "12px 16px",
+  large: "14px 28px",
+  small: "8px 12px",
+};
+
+const fontSizes = {
+  normal: "14px",
+  large: "16px",
+  small: "14px",
+};
+
+const bgColors = {
+  primary: "#00a991",
+  secondary: "#e8edf1",
+};
+
+const colors = {
```

Comment:
there is already `constants.js` file in master so you should use it instead of hardcoded.  Same for bgColors this colors should be in `constants.js`